msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationJavaTest.java

```diff
@@ -2654,9 +2650,9 @@ public void testSignUpEmptyUsernameNoException() throws ExecutionException, Inte
         SignUpResult signUpResult = signUpTestCallback.get();
 
         assertTrue(signUpResult instanceof SignUpError);
+        assertTrue(((SignUpError) signUpResult).isInvalidUsername());
```

Comment:
It looks like the assertion here is swapped from false to true - is that due to the old testing logic being incorrect?

Reply:
No, previously the error returned was UnknownError so checking it was not invalidUsername was correct.